Accordion.js

```diff
@@ -97,38 +99,51 @@ export default class Accordion extends Component {
       </Collapsible>
     );
 
+    const checkIfItemIsDisabled = key => {
+      if (
+        (typeof disabled === 'boolean' && disabled) ||
+        (Array.isArray(disabled) && disabled.includes(key))
+      ) {
+        return true;
+      }
+      return false;
+    };
```

Comment:
Think this could be simplified no?

```js
return (typeof disabled === 'boolean' && disabled) || (Array.isArray(disabled) && disabled.includes(key))
```